MvvmCross/Binding/Droid/Views/MvxAdapter.cs

```diff
@@ -307,4 +308,31 @@ protected virtual IMvxListItemView CreateBindableView(object dataContext, int te
             return new MvxListItemView(this._context, this._bindingContext.LayoutInflaterHolder, dataContext, templateId);
         }
     }
+
+	public class MvxAdapter<TItem> : MvxAdapter, IMvxAdapter where TItem : class
+	{
+		public MvxAdapter(Context context) : base(context, MvxAndroidBindingContextHelpers.Current())
+		{
+		}
+
+		public MvxAdapter(Context context, IMvxAndroidBindingContext bindingContext) : base(context, bindingContext)
+		{
+		}
+
+		public MvxAdapter(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
+		{
+		}
+
+		public new IList<TItem> ItemsSource
```

Comment:
Shouldn't this be `IEnumerable<TItem> ItemsSource`?  The base class returns `IEnumerable` and might not be able to cast to `IList<TItem>>`.